src/source/src/table.rs

```diff
@@ -67,75 +72,78 @@ impl TableDmlHandle {
         }
     }
 
-    pub fn stream_reader(&self) -> TableStreamReader {
+    pub fn stream_reader(&self, actor_id: ActorId) -> TableStreamReader {
         let mut core = self.core.write();
         let (tx, rx) = mpsc::channel(DML_CHUNK_BUFFER_SIZE);
```

Comment:
The `Begin`, `End`, and `Rollback` will also be counted here which I'm not sure is reasonable. From my perspective, I guess it's better to ignore these control messages, then it's also possible to synchronously send `Rollback` message in `Drop`.

Channels with customized permits sound like applicable here, but I'm unsure if it is overkill.

https://github.com/risingwavelabs/risingwave/blob/c3091f0f5a77d007e857d688f3cf4e3e831d237e/src/stream/src/executor/exchange/permit.rs#L98-L129